Modules/mmapmodule.c

```diff
@@ -1094,7 +1094,7 @@ mmap__repr__method(PyObject *op)
 
 #ifdef MS_WINDOWS
 static PyObject *
-mmap__sizeof__method(PyObject *op, void *Py_UNUSED(ignored))
+mmap__sizeof__method(PyObject *op, PyObject *Py_UNUSED(ignored))
```

Comment:
```suggestion
mmap__sizeof__method(PyObject *op, PyObject *Py_UNUSED(dummy))
```

Sorry, my bad. Since it was guarded by `MS_WINDOWS`, I missed the issue =/ (let's also rename ignored to dummy as it's a NOARG method)

Reply:
No worries - hard to spot! MSVC didn't complain, and clang-cl is quite niche.

Thanks a lot for all your work here!